AndelaOSP/Andela-Socials
# #165881843 (chore): Add featured image to slack

#### What Does This PR Do?
- adds a featured image in slack message

#### Description Of Task To Be Completed
- Add a featured image to the message sent to a user on slack when a new event is created

#### Any Background Context You Want To Provide?
- N/A

#### How can this be manually tested?
- `git fetch` and switch to the branch `ch-add-featured-image-165881843`
- create a new event and check message received on slack for the featured image

#### What are the relevant pivotal tracker stories?
[#165881843 Add featured image to slack](https://www.pivotaltracker.com/story/show/165881843)

#### Screenshot(s)
![image](https://user-images.githubusercontent.com/28973383/57375813-0fcbc380-71a7-11e9-8113-de398cc5d052.png)
